index.html

```diff
@@ -233,7 +233,32 @@ <h2 class="header landing__header header__about">
       </div>
     </div>
   </div>
-
   <!-- end of works -->
+
+  <!-- Contact Form  -->
+  <footer class="contact">
+    <h2 class="contact__header">Contact me</h2>
+    <p class="contact__text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/mdobpebd" method="post" class="contact__form">
+      <ul class="contact__form-list">
+        <li class="contact__form-list--item">
+          <label for="name"></label>
+          <input type="text" id="name" class="contact__form-input" maxlength="30" placeholder="Sirri" required>
```

Comment:
it would be better if you change the placeholder to  `placeholder="Full name"`  that way it will make sense